src/pretix/control/templates/pretixcontrol/user/2fa_confirm_totp.html

```diff
@@ -37,7 +37,38 @@
         </li>
         <li>
             {% trans "Add a new account to the app by scanning the following barcode:" %}
-            <div class="qrcode-canvas" data-qrdata="#qrdata"></div>
+          <div class="qrcode-canvas" data-qrdata="#qrdata"></div>
+            <p>
+              <a data-toggle="collapse" href="#no_scan" aria-expanded="false" aria-controls="collapseExample">
```

Comment:
Fixed in ba4b0c4 - wasn't sure how to use `aria-*` attributes, so I left them as is from the template... my bad!